todo_app/app.py

```diff
@@ -1,5 +1,7 @@
-from flask import Flask
-
+from flask.globals import request
```

Comment:
Minor, but worth noting that this and `render_template` can both also be imported directly from flask which is more standard